locales/base/translation.json

```diff
@@ -2412,6 +2412,7 @@
       "earnUpToLabel": "You could earn up to:",
       "rateLabel": "Rate (est.)",
       "earnUpTo": "~{{fiatSymbol}}{{amount}} / yr",
+      "earnUpToV2": "<0></0> / yr",
```

Comment:
should this be `v1_XX` to match the other updated versions of translations?

Reply:
Fixed in af0b259a889916e4d1efa01cde54d79fccdbf0ed!